### PR TITLE
[16.0][MIG] analytic_operating_unit: Migration to 16.0

### PR DESCRIPTION
**Pull request by BT-aleonard**

### CODE REVIEW

**Review by BT-anieto — APPROVED** [review]
Thx a lot!

**Review by BT-rmartin — APPROVED** [review]
Well done with the migration. 
By the way I see no unit tests in the module, that could be added once it's merged as an improvement

**Comment by BT-aleonard** [comment]
Hi @AaronHForgeFlow,
Can you take a look to this PR, please?
Thanks.

File: analytic_operating_unit/views/analytic_account_view.xml
Comment (AaronHForgeFlow) [inline analytic_operating_unit/views/analytic_account_view.xml]:
```suggestion
            <field name="currency_id" position="before">
```

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM, but I had to modify the form view in order to edit the operating units in runbot.

File: analytic_operating_unit/views/analytic_account_view.xml
Comment (AaronHForgeFlow) [reply analytic_operating_unit/views/analytic_account_view.xml]: @BT-aleonard would yo attend this commit? I will merge after that.

Comment (BT-aleonard) [reply analytic_operating_unit/views/analytic_account_view.xml]: It's done

Comment (AaronHForgeFlow) [reply analytic_operating_unit/views/analytic_account_view.xml]: Thanks

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**
